.gbp.conf

```diff
@@ -1,3 +0,0 @@
-[DEFAULT]
-debian-branch=debian
-upstream-branch=master
```

Comment:
This file would be needed since standard layout for gbp is to have master containing packaging etc